CONTRIBUTORS.md

```diff
@@ -63,6 +72,7 @@ The following individuals have contributed code to Galaxy:
 * Saskia Hiltemann <zazkia@gmail.com>
 * Rob Hooft <rob.hooft@nbic.nl>
 * Y. Hoogstrate <y.hoogstrate@erasmusmc.nl>
+* Hans-Rudolf Hotz <hrhotz@users.noreply.github.com>
```

Comment:
(turns out, we only had two with missing email addresses in their commits, pinged both)